### PR TITLE
Apply black style to test_Graphics* files, version 19.10b0.

### PR DESCRIPTION
**Pull request by svalqui**
This pull request addresses issue #2552 

- [X] I hereby agree to dual licence this and any previous contributions under both
the _Biopython License Agreement_ **AND** the _BSD 3-Clause License_.

- [X] I have read the ``CONTRIBUTING.rst`` file, have run ``flake8`` locally, and
understand that AppVeyor and TravisCI will be used to confirm the Biopython unit
tests and style checks pass with these changes.

- [X] I have added my name to the alphabetical contributors listings in the files
``NEWS.rst`` and ``CONTRIB.rst`` as part of this pull request, am listed
already, or do not wish to be listed. (*This acknowledgement is optional.*)

This PR applies black style to test_Graphics* files, files listed below; small changes should be fine to merge.
test_GraphicsBitmaps.py
test_GraphicsChromosome.py
test_GraphicsDistribution.py
test_GraphicsGeneral.py

### CODE REVIEW

**Comment by peterjc** [comment]
In ``Tests/test_GraphicsChromosome.py`` where you turn off black, does it want to put each three-tuple on its own line?

**Comment by svalqui** [comment]
Yes, it makes it a long list (vertical).

File: Tests/test_GraphicsChromosome.py [hunk 1]
```diff
@@ -246,10 +246,9 @@ def test_widget(self):
         properties = new_stdout.getvalue()
         sys.stdout = save_stdout
 
-        self.assertTrue(
-            expected_string in properties,
-            "Unexpected results from dumpProperties: \n %s" % properties,
-        )
```
Comment (peterjc) [inline Tests/test_GraphicsChromosome.py]: Was there a merge conflict? That looks to have lost the black formatting...

Comment (svalqui) [reply Tests/test_GraphicsChromosome.py]: Yes, a merge conflict, fixed and re applied black; I'll try to fix the merge conflicts for this issue before the commit freeze for the new release; we are in lock down I can only do a tiny bit at a time.

**Comment by peterjc** [comment]
Looks good, thanks!

I quite understand everyone's normal routines are disrupted at the moment, mine too.

**Pull request merged**